core/transfer/src/main/java/org/eclipse/dataspaceconnector/transfer/core/CoreTransferExtension.java

```diff
@@ -65,6 +65,10 @@
 
     @EdcSetting
     private static final String TRANSFER_STATE_MACHINE_BATCH_SIZE = "edc.transfer.state-machine.batch-size";
+    @EdcSetting
+    private static final String TRANSFER_SEND_RETRY_LIMIT = "edc.transfer.send.retry.limit";
```

Comment:
I don't really think so. In a long list of settings it would seem "edc.transfer.retry.limit" is the times it retries performing the data transfer itself, not sending it to the provider.